### ISSUE TITLE
Snowcapped Mountains using -* snow.gifs

### ISSUE THREAD

**Issue by BLOODWOLF333** [MegaMek/megamek#280]
http://megamek.info/forums/index.php?topic=2443.msg16464#new

Its going to look like blue icing on top of the mountains with having a snow theme and MM using the correct (positive) snow images for mountain tops compared to how it has been using the -1:-6 snow images somehow.  I dont want to overwrite someone else's images but.........snow isnt blue.

![mm0](https://cloud.githubusercontent.com/assets/16950832/16934787/6e549aee-4d1f-11e6-959a-0f1f28b3d0d0.png)

[snow theme.zip](https://github.com/MegaMek/megamek/files/370431/snow.theme.zip)

**Comment by Akjosch** [comment]
I'm not sure what is being asked here. Because if it's about reworking of the current themes and making things like snow look better, I'm all for it.

**Comment by BLOODWOLF333** [comment]
Yeah I already talked to Arlith about this on IRC.  The author of the MM forum post's concern was how currently MM puts black snow ontop of mountains and it looks bad.  So, I created a snow theme for our stock tilesets to use and the "above" is the end result.  MM's stock snow images are icing blue and dont look any better.  But, I dont want to just overwrite someone else's images in the stock snow folder and replace then with my more white ones.
